Json/index.ts

```diff
@@ -0,0 +1,33 @@
+import { isly } from "isly"
+import { Data as JsonData } from "./Data"
+
+export class Json<T extends Json.Data = Json.Data> {
+	constructor(public options: Json.Options<T> = { type: Json.Data.type as isly.Type<T> }) {}
```

Comment:
this will stop defaulting to the given `type` if some other option is passed down. Showcase on how to pass `type` as default no as long it is not provided in this [playground](https://www.typescriptlang.org/play/?#code/MYewdgziA2CmB0w4EMBOAKAlAbgFAEswAXWVAM2WFgAIB5AByP3AmoG9drqyQQAuahCKpCAczxcARmgFgArgFtJpCdWkAvAZN4oweAL65cZOWGBNw1ACawKc6EQCC0aOjbde1ALzUARL4AaNTRvagAGII1QimgIWCD9AQAFNCZkaAAeBgtIAD5Qtn1MAWzmSHZOalRYIjlUMHYPEEi0FvVqfQNjU3My61tkeyJaMFg3JtD-IPgZkEYy1kTqFNQ0zNKWfJ9C4uX8YABrLPmWIN8eEF98gDJl1Px0rIV8ImOciDOLq-yOLmra+qNC7TWYncqdXCGXCgSBEKqwCBDVjbSrpaACX5cagARgENjsDmcrkwAUqXAATHiBkMieMLgJfAALWAuS4dElk6gAZipBKcLjpvAZzNZgWCqAE2PJXLaAmEchoRVJXH0yuo4FgGM5uP6fJGYw5WOolN1gwc+vG0glOOl7LVXB5pqGFvc9L8Iuglxa1qlXLtlVVkLw0JYMAQntE6GqiIcEBwuCAA) just hit run to see logs.

Reply:
Fixed in rewrite.